nimbus/src/persistence.rs

```diff
@@ -15,14 +15,123 @@ use rkv::StoreOptions;
 use std::fs;
 use std::path::Path;
 
+// Stolen from glean - use a feature to choose between the backends.
```

Comment:
Glean freely gives ;)
```suggestion
// Inspired by glean - use a feature to choose between the backends.
```